src/libs/SidebarUtils.js

```diff
@@ -345,6 +350,11 @@ function getOptionData(report, reportActions, personalDetails, preferredLocale,
         });
     }
 
+    if (isMoneyRequest && visibleReportActionItems[report.reportID] && isDeletedParentAction) {
```

Comment:
Could you add a comment on this? I found it helpful to understand why we are doing this.